packages/frontend/app/components/learner-group/calendar.gjs

```diff
@@ -61,22 +62,24 @@ export default class LearnerGroupCalendarComponent extends Component {
     return await map(flat, async (offering) => {
       const session = await offering.session;
       const course = await session.course;
-      return {
-        startDate: offering.startDate.toISOString(),
-        endDate: offering.endDate.toISOString(),
-        calendarStartDate: offering.startDate.toISOString(),
-        calendarEndDate: offering.endDate.toISOString(),
-        courseTitle: course.title,
-        name: session.title,
-        offering: offering.id,
-        location: offering.location,
-        color: '#84c444',
-        prerequisites: [],
-        postrequisites: [],
-        isScheduled: session.isScheduled || course.isScheduled,
-        isPublished: session.isPublished && course.isPublished,
-        isBlanked: false,
-      };
+      const school = await course.school;
+      return this.schoolEvents.createEventFromData(
+        {
+          startDate: offering.startDate.toISOString(),
+          endDate: offering.endDate.toISOString(),
+          courseTitle: course.title,
+          school: school.id,
```

Comment:
for learner-group and offering calendars, i decided to create these pseudo-events as school events, seems the less-wrong context hack here. adding `school` for consistency, this attr is currently not processed (we don't create event slugs in these calendars).